ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/LedgerDaoContractsReader.scala

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.appendonlydao
```

Comment:
I find it very confusing that now `store/dao` as well as `store/appendonly` contain interfaces that are implemented in a criss-crossed manner in the two projects. I posit to create `store/interface` project and start moving interfaces there.

Reply:
Moved `LedgerDaoContractsReader` to `..store.interfaces` package. Not sold on this solution, but you have a point about the annoying dependency bisection here. I guess no solution is ideal, but it will be cleaned-up when we move to the append-only package.